river/command/set_repeat.zig

```diff
@@ -0,0 +1,42 @@
+// This file is part of river, a dynamic tiling wayland compositor.
+//
+// Copyright 2020 Bonicgamer
```

Comment:
Feel free to add yourself to AUTHORS if you would like credit, just pushed the changes I proposed on irc to master so this should be:
```suggestion
// Copyright 2020 The River Developers
```

Reply:
I've decided after sitting in front of my computer screen for about 30 minutes that I'm going to not add myself to the AUTHORS file since most of the code was copied from River.

Once when I make something without copying code then I will add myself to the AUTHORS file.